xml/System.Collections.Generic/HashSet`1.xml

```diff
@@ -2623,11 +2623,11 @@ The following example demonstrates how to merge two disparate sets. This example
         </Parameter>
       </Parameters>
       <Docs>
-        <typeparam name="TAlternate">To be added.</typeparam>
-        <param name="lookup">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="TAlternate">The alternate type of instance for performing lookups.</typeparam>
+        <param name="lookup">The created lookup instance when the method returns true, or a default instance that should not be used if the method returns false.</param>
+        <summary>Gets an instance of a type that may be used to perform operations on the current <see cref="T:System.Collections.Generic.HashSet`1"/> using a <typeparamref name="TAlternate"/> instead of a <typeparamref name="T"/>.</summary>
```

Comment:
```suggestion
        <summary>Gets an instance of a type that can be used to perform operations on the current <see cref="T:System.Collections.Generic.HashSet`1"/> using a <typeparamref name="TAlternate"/> instead of a <typeparamref name="T"/>.</summary>
```